src/libs/actions/IOU.js

```diff
@@ -101,7 +101,7 @@ function buildOnyxDataForMoneyRequest(
     const optimisticData = [
         {
             // Use SET for new reports because it doesn't exist yet, is faster and we need the data to be available when we navigate to the chat page
-            onyxMethod: isNewChatReport ? Onyx.METHOD.SET : Onyx.METHOD.MERGE,
+            onyxMethod: Onyx.METHOD.MERGE,
```

Comment:
These are being changed because:
1. Set and merge shouldn't be mixed. It will lead to undefined behavior of the order in which subscribers are updated (ie. you end up seeing a little flash of incorrect data as amounts go from $0 to whatever was set in the request).
2. Using `set` just because "it's faster" is not a reason to use set

I checked with @luacmartins who originally made the change and we agreed it was OK to use merge for everything for now.